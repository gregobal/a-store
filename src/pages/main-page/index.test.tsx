import {render} from "@testing-library/react";
import {BrowserRouter} from "react-router-dom";
import {MainPage} from "./index";

describe("Main Page", () => {
    it("snapshot", () => {
        const view = render(
            <BrowserRouter>
                <MainPage />
            </BrowserRouter>
        );

        expect(view).toMatchSnapshot();
    });
});
