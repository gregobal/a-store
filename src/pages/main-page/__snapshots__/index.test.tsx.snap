// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Page snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <article
        class="grid__row_1gt5e grid__component_xpkmx grid__center_xpkmx container"
      >
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-12_17rb3 grid__width-desktop-6_17rb3"
        >
          <a
            class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z container left"
            href="/sdelano-v-alfe"
          >
            <span
              class="link__text_1r33z"
            >
              <h1
                class="typography__component_h1nzy typography__styrene-medium_qohs3 typography__bold_h1nzy"
              >
                Сделано в Альфе
              </h1>
            </span>
          </a>
        </section>
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-12_17rb3 grid__width-desktop-6_17rb3"
        >
          <a
            class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z container right"
            href="/svoy-dizain"
          >
            <span
              class="link__text_1r33z"
            >
              <h1
                class="typography__component_h1nzy typography__styrene-medium_qohs3 typography__bold_h1nzy"
              >
                Свой дизайн
              </h1>
            </span>
          </a>
        </section>
      </article>
    </div>
  </body>,
  "container": <div>
    <article
      class="grid__row_1gt5e grid__component_xpkmx grid__center_xpkmx container"
    >
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-12_17rb3 grid__width-desktop-6_17rb3"
      >
        <a
          class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z container left"
          href="/sdelano-v-alfe"
        >
          <span
            class="link__text_1r33z"
          >
            <h1
              class="typography__component_h1nzy typography__styrene-medium_qohs3 typography__bold_h1nzy"
            >
              Сделано в Альфе
            </h1>
          </span>
        </a>
      </section>
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-12_17rb3 grid__width-desktop-6_17rb3"
      >
        <a
          class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z container right"
          href="/svoy-dizain"
        >
          <span
            class="link__text_1r33z"
          >
            <h1
              class="typography__component_h1nzy typography__styrene-medium_qohs3 typography__bold_h1nzy"
            >
              Свой дизайн
            </h1>
          </span>
        </a>
      </section>
    </article>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
